ci-operator/config/openshift/azure-storage-azcopy/openshift-azure-storage-azcopy-release-4.17.yaml

```diff
@@ -1,23 +1,23 @@
 base_images:
-  ocp_4.16_base:
+  ocp_4.17_base:
     name: "4.17"
     namespace: ocp
-    tag: base
-  ocp_builder_rhel-8-golang-1.20-openshift-4.16:
+    tag: base-rhel9
+  ocp_builder_rhel-9-golang-1.21-openshift-4.17:
     name: builder
     namespace: ocp
-    tag: rhel-8-golang-1.20-openshift-4.16
+    tag: rhel-9-golang-1.21-openshift-4.17
```

Comment:
pj-rehearse says this image does not exist.

I think you should keep 4.16 in the file as it was before, only bump it to RHEL9

Reply:
@jsafrane Interesting, then I don't understand the purpose of this config. Also, the version change got taken care of by automation:

For main:
https://github.com/openshift/release/commit/3b06c6790b798a97ee34f46e5cfb0d4d5e3ea5e2#diff-91d9d6a06bf13805114ebb0733044b30739ea1f833bca6f29785ab991aedbd80

For branches: 
https://github.com/openshift/release/commit/2091c1396758a4862236d53a81102bfaae16ce8a#diff-787c43ff0a75a87634925bde38dbd77dc8e8c4fc10d625416006de65a3cd69fa

Closing this PR - no longer needed.